forum/Sources/Subs.php

```diff
@@ -1599,6 +1599,33 @@ function parse_bbc($message, $smileys = true, $cache_id = '', $parse_tags = arra
 				'before' => '<span style="color: white;" class="bbc_color">',
 				'after' => '</span>',
 			),
+			array(
+				'tag' => 'video',
+				'type' => 'unparsed_content',
+				'content' => '<iframe width="928" height="522" src="https://www.youtube.com/embed/$1" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>',
```

Comment:
There should be a `max-width: 100%` style attribute on this, otherwise it forces the video to always be 928px wide, even on mobile.